service-api/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM php:8.3-fpm-alpine AS app
 
-RUN apk --no-cache add fcgi icu-dev $PHPIZE_DEPS \
+RUN apk --no-cache add fcgi icu-dev libxml2-dev $PHPIZE_DEPS \
   && docker-php-ext-install opcache \
+  && docker-php-ext-install soap \
   && docker-php-ext-enable sodium
-RUN pecl install apcu && docker-php-ext-enable apcu
+RUN pecl install apcu && docker-php-ext-enable apcu && docker-php-ext-enable soap
```

Comment:
Enabling the extension is done implicitly in `docker-php-ext-install`
```suggestion
RUN pecl install apcu && docker-php-ext-enable apcu
```

Reply:
Ah good shout, shows how long it's been since I did this.